packages/components-native/src/Checkbox/Checkbox.test.tsx

```diff
@@ -102,3 +104,118 @@ describe("Checkbox", () => {
     expect(getByTestId("minus2")).toBeDefined();
   });
 });
+
+describe("Checkbox with FormProvider", () => {
```

Comment:
Thanks for adding these tests. I am actually shocked we didn't have tests for Checkbox in a form. I thought we had tests for all our inputs in tests but I can't seem to find the checkbox one